lib/orbit_web/controllers/sign_in_controller.ex

```diff
@@ -8,6 +8,45 @@ defmodule OrbitWeb.SignInController do
 
   alias Orbit.Repo
 
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, params) do
+    line = params["line"] || "blue"
+
+    service_date =
+      if params["service_date"],
+        do: Date.from_iso8601!(params["service_date"]),
+        else: Util.Time.service_date_for_timestamp(Util.Time.current_time())
+
+    {start_datetime, end_datetime} = Util.Time.service_date_boundaries(service_date)
+
+    rail_line = String.to_existing_atom(line)
+
+    json(
+      conn,
+      %{
+        data:
+          Repo.all(
+            from(si in OperatorSignIn,
+              where:
+                ^start_datetime < si.signed_in_at and
+                  si.signed_in_at < ^end_datetime and
```

Comment:
**nitpick:** I think that one or both of these inequalities should be inclusive, just in case we ever hit the edge case where someone is signed in during the second in which the service date rolls over.

Reply:
I'd prefer not to change it. It's unambiguous that < 3:00 is the previous service day, and it's easily explainable that way. Further, sign-ins are anecdotally rarely < 4:00

Reply:
Right, < 3:00 is the previous service day, but exactly 3:00 is the current service day, and with the code as-written 3:00 on the dot doesn't fall into either service date. I agree that it's highly unlikely to ever be a problem in practice, but I think theoretically at least it's incorrect.

Reply:
OH! Gotcha. Yep, I agree. Will fix 👍

Reply:
6e54d41